src/main/resources/assets/computercraft/lua/rom/programs/lua.lua

```diff
@@ -19,11 +19,13 @@ local tEnv = {
 setmetatable( tEnv, { __index = _ENV } )
 
 if term.isColour() then
-    term.setTextColour( colours.yellow )
+    term.setTextColour( colors[ settings.get( "lua.prompt_color" ) ] or colors.yellow)
+else
+    term.setTextColour( colors[ settings.get( "lua.prompt_color" ) ] or colors.white)
 end
 print( "Interactive Lua prompt." )
 print( "Call exit() to exit." )
-term.setTextColour( colours.white )
+term.setTextColour( colors[ settings.get( "lua.text_color" ) ] or colors.white )
 
 while bRunning do
     --if term.isColour() then
```

Comment:
This should be reimplemented (using `lua.prompt_color`, not sure why this was removed in the first place)